dev-packages/test-utils/src/playwright-config.ts

```diff
@@ -22,7 +22,7 @@ export function getPlaywrightConfig(
   const config: PlaywrightTestConfig = {
     testDir: './tests',
     /* Maximum time one test can run for. */
-    timeout: 150_000,
```

Comment:
We used to have a very long timeout here because we used to poll for events on sentry and that could take a while. Nowadays the tests that poll sentry have their own playwright configs with long timeouts so we can lessen the timeout here.